versions.props

```diff
@@ -6,7 +6,7 @@ org.apache.hadoop:* = 2.7.3
 org.apache.hive:* = 2.3.8
 org.apache.httpcomponents.client5:* = 5.1
 org.apache.orc:* = 1.7.5
-org.apache.parquet:* = 1.12.2
+org.apache.parquet:* = 1.12.3
```

Comment:
Looks like this is the only necessary change. Can you update this PR to just this change, or can you open a separate PR for this?

Reply:
Yeah absolutely.

This PR was originally for running the JMH tests.

I’ll remove all changes but the version bump and consider opening a separate PR for them if we want them.